lib/jq/rhmi-usage-common.jq

```diff
@@ -0,0 +1,129 @@
+module {
+  "name": "rhmi-usage-common",
+  "description": "shared functions used by rhmi-usage-report-* scripts",
+  "homepage": "https://github.com/integr8ly/rhmi-utils#readme",
+  "license": "Apache",
+  "author": "Jesse Sarnovsky",
+  "version": "1.0.0",
+  "jq": "1.6",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/integr8ly/rhmi-utils.git"
+  }
+};
+
+import "i8-helpers" as i8;
+
+def getNamespaceClasses:
+  [
+    .namespaces[] | {
+      ns: .metadata.name,
+      labels: .metadata.labels
+    } | {
+      ns,
+      nsClass: (
+        if .labels.rhmi=="true" or .labels.integreatly=="true" then
+          "rhmi"
+        elif .ns=="redhat-rhmi-operator" then
+          "rhmi"
```

Comment:
Could just check for `redhat-rhmi` here to simplify.

Reply:
I didn't want to rely on naming because these scripts are intended to be usable against rhmi 1.x and 2.x (pds, osd, and theoretically any ocp w/ rhmi installed)

Reply:
That makes sense, thanks @jessesarn